incubator/hnc/Makefile

```diff
@@ -185,10 +185,14 @@ kind-deploy:
 
 .PHONY: e2e-test
 e2e-test: deploy
+	go clean -testcache
+	sleep 15
```

Comment:
Document where these numbers came from, and (more importantly) why they're different between the local and remote versions.

Reply:
The numbers come from multiple experiments. I'm not sure why webhook server on kind takes more time.